docs/kratos/organizations/organizations.mdx

```diff
@@ -3,6 +3,14 @@ id: organizations
 title: Organizations and B2B Single Sign-On
 ---
 
+:::info
```

Comment:
Does this really need to be at the very top of this document, highlighted? Maybe add it as a section at the end?

Reply:
ping @vinckr